gapis/api/gles/api/extensions.api

```diff
@@ -642,9 +645,12 @@ cmd void glEGLImageTargetTexture2DOES(GLenum target, GLeglImageOES image) {
     glErrorInvalidOperation_ObjectDoesNotExist!GLeglImageOES(image)
   }
   t := GetBoundTextureOrErrorInvalidEnum(target)
-  SetTextureImage(t, 0, 0, eglImage.Image)
-  t.EGLImage = eglImage
-  GetEGLImageData(eglImage.ID, eglImage.Image.Width, eglImage.Image.Height)
+
+  for _, l, img in eglImage.Images {
+    SetTextureImage(t, 0, as!GLint(l), img)
+    t.EGLImage = eglImage
+    GetEGLImageData(eglImage.ID, img.Width, img.Height)
```

Comment:
You probably don't want to call this n times...

Reply:
Added a `if l == 0` block around these two statements (we still need the image width / height from somewhere)